scripts/create-components-images-files.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+ set -e
+
+cd $(dirname $0)
+
+source version
+source export-config
+source package-env
+
+cd ../package
+
+../scripts/k3s-images.sh
+
+# Make sure the used data.json is a release artifact
+
+if [ ${ARCH} == arm64 ]; then
+    ETCD_UNSUPPORTED_ARCH=arm64
+fi
+if [ ${ARCH} == s390x ]; then
+    ETCD_UNSUPPORTED_ARCH=s390x
+fi
```

Comment:
No major deal, but I believe that this could be removed given that we no longer build for s390x.